tests/dry-run-kustomize.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -ex
+
+ENVIRONMENT=$1
+CLUSTER=$2
+CURRENT_DIRECTORY=$(pwd)
+ASO_VERSION=$(yq eval '.spec.chart.spec.version' apps/azureserviceoperator-system/aso/aso.yaml)
+kubeconform_config=("-strict" "-summary" "-n" "12" "-schema-location" "default" "-schema-location" "/tmp/schemas/$ENVIRONMENT/$CLUSTER/")
+ASO_URL="https://github.com/Azure/azure-service-operator/releases/download/"$ASO_VERSION"/azureserviceoperator_"$ASO_VERSION".yaml"
+CSI_URL="https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/deploy/secrets-store.csi.x-k8s.io_secretproviderclasses.yaml"
+
```

Comment:
All CRDs are taken from result of Kustomize output, but csi is enabled as a plugin and aso is using helm chart(so cannot extract CRDs)